platform/jewel/markdown/core/src/main/kotlin/org/jetbrains/jewel/markdown/scrolling/ScrollingSynchronizer.kt

```diff
@@ -124,15 +129,24 @@ public abstract class ScrollingSynchronizer {
             }
     }
 
+    @GenerateDataFunctions
```

Comment:
I think we don't use this annotation any more, you need to generate the equals/hashcode